.bazelrc

```diff
@@ -17,7 +17,7 @@
 
 # try-import /opt/credentials/bazel-remote-cache.rc
 
-build --workspace_status_command bin/workspace_status.sh --incompatible_strict_action_env --java_language_version=11 --javacopt='--release 11'
+build --incompatible_strict_action_env --java_language_version=11 --javacopt='--release 11' --@io_bazel_rules_docker//transitions:enable=false
```

Comment:
`--workspace_status_command bin/workspace_status.sh` is not needed any more as rules_docker now has a direct way of passing container tag names.

`--@io_bazel_rules_docker//transitions:enable=false` disables a feature that was implemented in rules_docker 0.23, that introduced cross-platform compilations. Unfortunately, this feature has broken non-cross-platform programming language Docker image builds on MacOS machines.